.Rbuildignore

```diff
@@ -1,9 +1,11 @@
 ^phsmethods\.Rproj$
 ^\.Rproj\.user$
 ^README\.Rmd$
+^README\.html$
```

Comment:
This shouldn't be needed now.
```suggestion
```

Reply:
Can we leave it there just in case?

Reply:
Yep, fine to leave in, it would just be redundant but won't negatively affect anything.